src/text_shaper/directwrite_shaper.cpp

```diff
@@ -671,42 +522,83 @@ std::optional<rasterized_glyph> directwrite_shaper::rasterize(glyph_key _glyph,
 
     auto const width = output.size.width;
     auto const height = output.size.height;
```

Comment:
```suggestion
    auto const [width, height] = output.size;
```
since you use `auto` type deduction, using tuple decomposition to get straight access to the members isn't too far off. What do you think? :-)